server/rosetta/client_online.go

```diff
@@ -6,34 +6,29 @@ import (
 	"encoding/hex"
 	"fmt"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/cosmos/cosmos-sdk/version"
 
 	abcitypes "github.com/tendermint/tendermint/abci/types"
 
-	"github.com/tendermint/btcd/btcec"
-
-	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
-
-	"github.com/coinbase/rosetta-sdk-go/types"
+	rosettatypes "github.com/coinbase/rosetta-sdk-go/types"
```

Comment:
imports are not ordered correctly

Reply:
goimports does not give warnings, nor edit the files if I do run it against the rosetta dir. 

do you use some other tooling?

Reply:
Yes, `goimports` is not able to reorder lines that are blank-separated. It's a known limitation. We can fix this in a separate PR